client/shipping-label/views/steps/address/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import FoldableCard from 'components/foldable-card';
+import { translate as __ } from 'lib/mixins/i18n';
+import AddressFields from './fields';
+import ExpandButton from 'shipping-label/views/expand-button';
+
+const Origin = ( props ) => {
+	return (
+		<FoldableCard
+			header={ __( 'Origin address' ) }
+			summary={ __( '' ) }
```

Comment:
Yep, I left this here as a placeholder. According to the latest mock-ups, the summaries will have plenty of content :)